Pod/Classes/UIFont+ArtsyFonts.m

```diff
@@ -105,10 +105,10 @@ + (UIFont *)serifItalicFontWithSize:(CGFloat)size
 + (UIFont *)sansSerifFontWithSize:(CGFloat)size
 {
     static dispatch_once_t onceToken;
-    NSString *fontFilename = useClosedFonts ? @"ITCAvantGardeDemi_Track03" : @"texgyreadventor-regular";
+    NSString *fontFilename = useClosedFonts ? @"AVG65lig" : @"texgyreadventor-regular";
 
-    NSString *type = useClosedFonts ? @"ttf" : @"ttf";
-    return [self ar_LoadAndReturnFont:@"AvantGardeGothicITCW01Dm" extension:type size:size onceToken:&onceToken fontFileName:fontFilename];
+    NSString *type = useClosedFonts ? @"otf" : @"ttf";
+    return [self ar_LoadAndReturnFont:@"AvantGardeGothicITC" extension:type size:size onceToken:&onceToken fontFileName:fontFilename];
```

Comment:
Do these font names still work with the OSS font?

Reply:
Yeah, I manually changed its metadata.

Reply:
It worked with just the .ttf file changed, but I had forgotten to also change the .bf file in `Pod/FontFiles` so that is updated now too.